docs/source/markdown/podman-network-create.1.md

```diff
@@ -38,6 +43,10 @@ Restrict external access of this network
 Allocate container IP from a range.  The range must be a complete subnet and in CIDR notation.  The *ip-range* option
 must be used with a *subnet* option.
 
+**--macvlan**
+
+Create a *macvlan* based connection rather than a classic bridge based on the given host-device.
```

Comment:
This is a little unclear - "based on the given host device" could apply to "classic bridge" or "macvlan based connection"